### PR TITLE
Chat 33 button

### PR DESCRIPTION
**Pull request by DStukalo**

### CODE REVIEW

File: src/ui-kit/components/Button/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  isStartIcon = false,
+  isDisabled = false,
+  isEndIcon = false,
+  label = 'Button',
+  handleClick,
+  ariaLabel = ' ',
+}) {
+  if (isOnlyIcon)
+    return (
+      <IconButton
```
Comment (FortizFil) [inline src/ui-kit/components/Button/Button.jsx:17]: This will be another component, remove this

File: src/ui-kit/components/Button/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        css={buttonCss}
+        endIcon={isEndIcon}
+        onClick={handleClick}
+        disabled={isDisabled}
+      >
+        {label}
+      </Button>
+    );
+  return (
+    <Button css={buttonCss} onClick={handleClick} disabled={isDisabled}>
```
Comment (FortizFil) [inline src/ui-kit/components/Button/Button.jsx:50]: You should return one universal component, not different ones for each case

File: src/ui-kit/components/Button/Button.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      {label}
+    </Button>
+  );
+}
+
+// DefaultButton.defaultProps = {
```
Comment (FortizFil) [inline src/ui-kit/components/Button/Button.jsx:56]: What is it for?

File: src/ui-kit/components/Button/Button.styled.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/react';
```
Comment (FortizFil) [inline src/ui-kit/components/Button/Button.styled.js:1]: Why do you use this, I told to use styled

**Review by FortizFil — CHANGES_REQUESTED** [review]

File: src/ui-kit/components/Button/index.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import DefaultButton from './Button';
```
Comment (FortizFil) [inline src/ui-kit/components/Button/index.js:1]: We do it through the default export, it should look like this: export {default} from ....

File: src/ui-kit/components/Button/Button.jsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+/** @jsxImportSource @emotion/react */
+import { StyledButton } from './Button.styled';
+import { memo } from 'react';
+
+function DefaultButton({
+  startIcon = null,
+  isDisabled = false,
+  endIcon = false,
```
Comment (FortizFil) [inline src/ui-kit/components/Button/Button.jsx:8]: Should be null

File: src/ui-kit/components/Button/Button.styled.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    background: ${(p) => p.theme.palette.primary.main};
+    border: 1px solid ${(p) => p.theme.palette.primary.contrastText};
+  }
+
+  &:disabled {
+    border: 1px solid ${(p) => p.theme.palette.primary.light};
+    background: ${(p) => p.theme.palette.primary.disabled};
+  }
+
+  .MuiButton-startIcon {
```
Comment (FortizFil) [inline src/ui-kit/components/Button/Button.styled.js:34]: Nothing needs to be changed for the endIcon?

File: src/ui-kit/components/Button/Button.jsx [hunk 8]
```diff
@@ -0,0 +1,26 @@
+/** @jsxImportSource @emotion/react */
```
Comment (FortizFil) [inline src/ui-kit/components/Button/Button.jsx:1]: What is it for?

File: src/ui-kit/components/Button/Button.jsx [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+import { StyledButton } from './Button.styled';
+import { memo } from 'react';
+
+function DefaultButton({
+  startIcon = null,
+  isDisabled = false,
+  endIcon = false,
+  label = 'Button',
+  handleClick,
+  ...props
```
Comment (FortizFil) [inline src/ui-kit/components/Button/Button.jsx:11]: Please change structure: label, click, isDisabled, startIcon, endIcon, withoutBackground, fontSize ...props
And add withoutBackground, fontSize

File: src/ui-kit/components/Button/Button.jsx [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+}) {
+  return (
+    <StyledButton
+      onClick={handleClick}
+      disabled={isDisabled}
+      startIcon={startIcon}
+      endIcon={endIcon}
+      {...props}
```
Comment (FortizFil) [inline src/ui-kit/components/Button/Button.jsx:19]: Add withoutBackground, fontSize

File: src/ui-kit/components/Button/Button.styled.js [hunk 11]
```diff
@@ -0,0 +1,38 @@
+import Button from '@mui/material/Button';
+import styled from '@emotion/styled';
+
+export const StyledButton = styled(Button)`
+  padding: 8px 12px;
+  text-transform: none;
+  font-size: ${(p) => (p.fontSize ? p.fontSize : '16px')};
```
Comment (FortizFil) [inline src/ui-kit/components/Button/Button.styled.js:7]: Maybe it should be  `${p.fontSize}px`

**Review by FortizFil — CHANGES_REQUESTED** [review]

File: src/ui-kit/components/Button/Button.styled.js (hunk 11 above)
Comment (FortizFil) [reply src/ui-kit/components/Button/Button.styled.js:7]: No, I just asked what the idea was.

Comment (DStukalo) [reply src/ui-kit/components/Button/Button.styled.js:7]: I think we pass value with measurement, but it maybe better to always use px

**Review by FortizFil — APPROVED** [review]

**Pull request merged**